tests/devices/default_qubit/test_default_qubit_native_mcm.py

```diff
@@ -424,156 +424,224 @@ def circuit(x):
         _ = circuit([0.1, 0.2])
 
 
-# pylint: disable=not-an-iterable
+# pylint: disable=import-outside-toplevel, not-an-iterable
 @pytest.mark.jax
-@pytest.mark.parametrize("mcm_method", ["one-shot", "tree-traversal"])
-@pytest.mark.parametrize("shots", [100, [100, 101], [100, 100, 101]])
-@pytest.mark.parametrize("postselect", [None, 0, 1])
-def test_sample_with_prng_key(mcm_method, shots, postselect):
-    """Test that setting a PRNGKey gives the expected behaviour. With separate calls
-    to DefaultQubit.execute, the same results are expected when using a PRNGKey"""
-    # pylint: disable=import-outside-toplevel
-    from jax.random import PRNGKey
-
-    dev = get_device(shots=shots, seed=PRNGKey(678))
-    params = [np.pi / 4, np.pi / 3]
-    obs = qml.PauliZ(0) @ qml.PauliZ(1)
-
-    def func(x, y):
-        obs_tape(x, y, None, postselect=postselect)
-        return qml.sample(op=obs)
-
-    func0 = qml.QNode(func, dev, mcm_method=mcm_method)
-    results0 = func0(*params)
-    results1 = qml.QNode(func, dev, mcm_method="deferred")(*params)
-
-    mcm_utils.validate_measurements(qml.sample, shots, results1, results0, batch_size=None)
+class TestJaxIntegration:
+    """Integration tests for dynamic_one_shot with jax"""
+
+    @pytest.mark.parametrize("mcm_method", ["one-shot", "tree-traversal"])
+    @pytest.mark.parametrize("shots", [100, [100, 101], [100, 100, 101]])
+    @pytest.mark.parametrize("postselect", [None, 0, 1])
+    def test_sample_with_prng_key(self, mcm_method, shots, postselect):
```

Comment:
This test and the one below it have just been moved into the `TestJaxIntegration` class so no need to be thorough reviewing the aforementioned two tests